glib/src/any_gobject.rs

```diff
@@ -0,0 +1,108 @@
+use crate as glib;
+use crate::subclass::prelude::*;
+use crate::{object_subclass, wrapper, Object};
+use std::any::Any;
+use std::cell::{Ref, RefCell, RefMut};
+
+mod imp {
+    use super::*;
+
+    #[derive(Debug)]
+    pub struct AnyGObject {
+        pub item: RefCell<Option<Box<dyn Any>>>,
+    }
+    impl Default for AnyGObject {
+        fn default() -> Self {
+            Self {
+                item: RefCell::new(None),
+            }
+        }
+    }
+    #[object_subclass]
+    impl ObjectSubclass for AnyGObject {
+        const NAME: &'static str = "AnyGObject";
+        type Type = super::AnyGObject;
+        type ParentType = Object;
+    }
+    impl ObjectImpl for AnyGObject {}
+}
+
+wrapper! {
+    pub struct AnyGObject(ObjectSubclass<imp::AnyGObject>);
```

Comment:
```suggestion
    pub struct BoxedAnyObject(ObjectSubclass<imp::AnyGObject>);
```

Etc IMHO. @bilelmoussaoui ?

Reply:
Hmm, not sure :) but i'm pretty bad at naming stuff! Maybe just BoxedObject?